site/content/docs/5.2/forms/input-group.md

```diff
@@ -101,6 +108,13 @@ Place any checkbox or radio option within an input group's addon instead of text
 
 While multiple `<input>`s are supported visually, validation styles are only available for input groups with a single `<input>`.
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This form variant can not be used with this design because it does not respect the Orange Design System specifications.
+
+For the correct design, please refer to unitary [Forms](https://system.design.orange.com/0c1af118d/p/88ab5b-forms/b/599459) components and [Pages](https://system.design.orange.com/0c1af118d/p/20500e-form/b/16bb53) form examples on  the Orange Design System website.
+{{< /ods-incompatibility-alert >}}
```

Comment:
Add a callout for #Sizing 
#Multiple addons : see with design because the placeholder is not suitable for accessibility
#Button addons: see with design what is ok or not => add a callout where needed